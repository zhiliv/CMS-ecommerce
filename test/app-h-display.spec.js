import { shallowMount } from '@vue/test-utils'
import H from '@/components/app/h-display/h-display.vue'

describe('app-h-display.js', () => {
  let cmp

  beforeEach(() => {
    cmp = shallowMount(H, {
      // Создать поверхностный экземпляр компонента
      propsData: {
        classes: 'simple simple-xl',
        styles: 'background: white',
        id: 'indent',
        level: '6',
        size: 4
      },
      slots:{
        default: 'Содержимое'
      }
    })
  })
  it('Содержимое html для app-h-display ', () => {
    expect(cmp.element).toMatchSnapshot()
  })
})
