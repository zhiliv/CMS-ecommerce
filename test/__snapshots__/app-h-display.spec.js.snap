// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`app-h-display.js Содержимое html для app-h-display  1`] = `
<h6
  class=" display-4 simple simple-xl"
  id="indent"
  style="background: white"
>
  Содержимое
</h6>
`;
